src/clarity-angular/utils/focus-trap/focus-trap.directive.ts

```diff
@@ -19,7 +22,16 @@ export class FocusTrapDirective implements AfterViewInit {
     }
 
     ngAfterViewInit() {
+        this._previousActiveElement = <HTMLElement>document.activeElement;
+        // console.log("Focus trap directive view init, activeElement: ", document.activeElement);
+        // console.log("Focus trap directive _previousActiveElement: ", this._previousActiveElement);
         const nativeElement: HTMLElement = this.elementRef.nativeElement;
         nativeElement.setAttribute("tabindex", "0");
     }
+
+    public setPreviousFocus(): void {
```

Comment:
Can you please add:
```typescript
ngOnDestroy() {
    this.setPreviousFocus();
}
```

It doesn't cost much and adds safety for other focus traps in the future that are not modals.

Reply:
Will add.